[DOCS] native realm needs to be explicitly disabled (#92279)
elastic/elasticsearch: x-pack/docs/en/security/authentication/configuring-native-realm.asciidoc

> The native realm is available and enabled by default. If you want to use a different method to manage and authenticate users you need to disable the native realm. 
> 
> The following snippet displays an example of how to disable the `native` realm.

Reviewer: This does not read right to me. We don't need disable native realm to use "a different method to manage and authenticate users". They can co-exist. I think we probably just need to say

```suggestion
The native realm is available and enabled by default. You can disable it explicitly with the following snippet. 
```

[source, yaml]
------------------------------------------------------------
xpack.security.authc.realms:
  native.native1:

>       order: 0

Reviewer: Nit: I don't particularly like the existing style because it is a bit too hierarchical and verbose. I think the more compact style looks better. But I have a few comments:
* Should we prefer it in future more consistently in other places? We don't need to touch anything else in this PR. But it would be good to know the future direction.
* Should we place `native.native1` on the same line as `xpack.security.authc.realms`? I think it is even more concise and direct to the context?
* The `order` field has a different amount of indentation.

Author: I think we should prefer it in future and make it more consistent in other places. I'll place `native.native1` onto the same line as `xpack.security.authc.realms` and fix the indentation issue with `order`. Thanks for catching that!
